### PR TITLE
New branch (journal-week-2) and journal outline

### PR DESCRIPTION
**Pull request by sjoyal**
@gatorpazz and @mstaehling can you review this badboy???

### CODE REVIEW

**Comment by gatorpazz** [comment]
Your syntax and prose are to die for. Good use of the word shit, 10/10 would read again :+1:

**Comment by MStaehling** [comment]
Solid outline :+1:

File: TIY_Week_2-Treading_Water_Barely.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    * QuotesRotator :poop:
+
+**Tips for surviving week 2**
+  * Looking forward to failure
+  * Growth mindset
+  * Sleep
+  * Keep reading and learning core concepts
+  * Don't bulldog the bad shit
+  * Journal entries aren't terrible, they just lag a week
+  * Beer is a hell of a drug
```
Comment (al-the-x) [inline TIY_Week_2-Treading_Water_Barely.md]: I'm Rick James Bitch!

File: about/cascading-style-sheets.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# About Cascading Style Sheets
+
+## Properties
+
+#### display
```
Comment (al-the-x) [inline about/cascading-style-sheets.md:13]: How 'bout those backticks?

File: about/cascading-style-sheets.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Usually block or inline, every element has a default display value.
+  * block - block-level elements starts on a new line and stretch left to right as far as they can. Example:
+    `<div>, <p>, <form>`
+  * inline - inline elements can wrap some text inside a paragraph without disrupting flow. Example:
+    `<span>, <a>`
+  * none - default value of JavaScript elements, used to hide and show elements without deleting and recreating them
+    * Different from visibility:hidden, which hides the element but still takes up the space it would if visible
```
Comment (al-the-x) [inline about/cascading-style-sheets.md:20]: Probably want to cover more of these... Specifically, I'm looking for `inline` and `inline-block`...

File: about/cascading-style-sheets.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+#### inline-block
+Creates a grid of boxes that fills the browser width and wraps nicely; easier approach than float. Can be used for layouts but must keep in mind the elements are affected by 'vertical-align'.
+  * Set the width of each column
+  * If there is whitespace between columns in HTML, there will be a gap between columns
+  Example:
+  `nav {
+      display: inner-block;
+      vertical-align: top;
+      width: 25%;
+    }`
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Might want to take a careful look at this...

Not sure you got this...

**Comment by JessyRiordan** [comment]
:+1:  Hilarious! I think this sums up exactly what we are all thinking!

**Comment by gatorpazz** [comment]
Coffee and tea are trade offs, it's an either or waste excretion. The sentiments are shared with the difficulty spike of week 2 and week 3 doesn't look like it'll be any better. 10/10, would read again :+1:

File: TIY_Week_2-Treading_Water_Barely.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+## TIY Week 2 - Treading Water Barely
+
+##### What the effffff just happened?
+
+Emphasis on the **F**. I just read my entry for week 1. It sounds remarkably upbeat and almost giddy with excitement. Who wrote that? All the things I was so positive about got rocked in the worst possible way during week 2. Git and github? el.oh.el. Practically everything I thought I was grasping turned out to be a lie. How was I supposed to know what a merge conflicts was? The fearless leader threw us into that fire firsthand. Learn by doing, he says. At least that means I followed directions properly since I avoided the problem until the intended face smack on Tuesday. And the JavaScript? Learning the fundamentals was borderline fun in week one. Applying those fundamentals to elements in the DOM? Not so much. I still have the command line to grasp onto. That didn't take an unexpected turn this week. Still, kind of sad. At least we have in class homework review. Where we all get to see how we're equally _dum_.
```
Comment (jorgehjr84) [inline TIY_Week_2-Treading_Water_Barely.md]: Six lines down, I believe it should be conflict. 

Overall, awesome read, kind of depressing, but in a funny way.

**Comment by jorgehjr84** [comment]
I thought I was the only one wondering how we stacked up to other cohorts. I always think, "are we behind." I read your first draft and definitely like the additions you made. :100:  points. Although points don't matter.

File: TIY_Week_2-Treading_Water_Barely.md (hunk 5 above)
Comment (gatorpazz) [inline TIY_Week_2-Treading_Water_Barely.md]: At the end of your first paragraph it's *dumb

**Comment by gatorpazz** [comment]
Not getting sarcasm aside good second week, onto the third! :+1:

**Comment by pope410211** [comment]
I enjoyed your journal! I also wonder how we compare to past cohorts and enjoyed your use of emojis! :+1:

**Comment by BFGriffith** [comment]
Nice summary of the emotional roller-coaster of week two! :+1:  :scream:

**Pull request merged**